lib/PuppeteerSharp.Tests/Tracing/TracingTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Tracing
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class TracingTests : PuppeteerPageBaseTest
+    {
+        private string _file;
+
+        public TracingTests()
+        {
+            _file = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+        }
+
+        public override void Dispose()
+        {
+            base.Dispose();
+
+            int attempts = 0;
+            const int attemptTimes = 5;
+
+            while (true)
+            {
+                try
+                {
+                    attempts++;
+                    if (File.Exists(_file))
+                    {
+                        File.Delete(_file);
+                    }
+                    break;
+                }
+                catch (UnauthorizedAccessException)
+                {
+                    if (attempts == attemptTimes)
+                    {
+                        break;
+                    }
+
+                    Task.Delay(1000).GetAwaiter().GetResult();
```

Comment:
This feels a bit weird, you should override `DisposeAsync` and in it call `await base.DisposeAsync()`